README.md

```diff
@@ -6,151 +6,107 @@ This repository holds the kcp tests that tests against the publicly available  i
 * Golang installed. See [Installing Golang](https://golang.org/doc/install),the newer the better.
         * Ensure you install Golang from a binary release [found here](https://golang.org/dl/), not with a package manager such as `dnf`
 * golint installed. See [Installing golint](https://github.com/golang/lint#installation)
-* Have the environment variable `KUBECONFIG` set pointing to your cluster
-
-### Include new test folder
-If you create a new folder for your test case, **add the path** to the [include.go](https://github.com/openshift/openshift-tests-private/blob/master/test/extended/include.go)
-
-### Create go-bindata for new YAML files
-If you have some **new YAML files** used in your code, you have to generate the bindata first.
-Run `make update` to update the bindata. For example, you can see the bindata has been updated after running the `make update` as follows:
-```console
-$ git status
-	modified:   test/extended/testdata/bindata.go
-	new file:   test/extended/testdata/olm/etcd-subscription-manual.yaml
-```
-
-### Compile the executable binary
-Note that we use the `go module` for package management, the previous `go path` is deprecated.
-```console
-$ git clone git@github.com:openshift/openshift-tests-private.git
-$ cd openshift-tests-private/
-$ make build
-mkdir -p "bin"
-export GO111MODULE="on" && export GOFLAGS="" && go build -o "bin" "./cmd/extended-platform-tests"
-$ ls -hl ./bin/extended-platform-tests 
--rwxrwxr-x. 1 cloud-user cloud-user 165M Jun 24 22:17 ./bin/extended-platform-tests
-```
-
+* Install [kubelogin](https://github.com/int128/kubelogin.git) and [KCP kubectl plugin](https://github.com/kcp-dev/kcp.git)
+      * Ensure your installed KCP kubectl plugin version is the same with the kcp service version, you could build the plugin by your self or just download the suitable version binary from [kcp release assets](https://github.com/kcp-dev/kcp/releases)
+  ```
```

Comment:
@Phaow also may be two different sections, if you would like to build please follow the below procedure. If you woul dlike to download the suitable version binary go to this page, download the zip file suitable for your OS, extract and copy the binaries to a place which is in your PATH and once copied run /usr/bin/kubectl-kcp --version to verify if server matches with client ? Can we include this information ?

Reply:
Sure, addressed.